src/modules/operations.js

```diff
@@ -98,6 +99,30 @@ const actions = {
 
 const getters = {
   getOperations: state => state.list,
+  getActiveOrders: state => {
+    const openOrders = state.list.filter(x => x.type === 'limit_order_create');
+    const cancelOrders = state.list.filter(x => x.type === 'limit_order_cancel');
+    const filledOrders = state.list.filter(x => x.type === 'fill_order');
+    const notCanceledOrders = openOrders.filter(x => !cancelOrders.some(y => y.orderId === x.orderId));
+    notCanceledOrders.forEach(notCancelOrder => {
+      let percentFilled = 0;
+      filledOrders.forEach(filledOrder => {
+        if (filledOrder.orderId === notCancelOrder.orderId) {
+          const remain = notCancelOrder.payload.amount_to_sell.amount - filledOrder.payload.pays.amount;
```

Comment:
Shouldn't we check if it is buy or sell order?

Reply:
Yes, it's work same for both variants